scripts/coursera_quiz_conversion.R

```diff
@@ -0,0 +1,79 @@
+# Need here and R.utils
+# For testing use tx_i  <- readLines(here("quizzes", "quiz_ch1.md"))
+
+# This is functionalized but running like script for now till we decide where to put this:
+# run by  pasting `Rscript scripts/coursera_quiz_conversion.R` in the terminal
+
+library(here)
+library(R.utils)
+
+
+coursera_quizzes <- function(path= here("quizzes"), # we might want to update to manuscript
```

Comment:
So you don't have to load a whole package for using one function, this would be a good spot to just use the `::`
```suggestion
coursera_quizzes <- function(path= here::here("quizzes"), # we might want to update to manuscript
```

Reply:
yeah

> This is awesome! I'm super glad you've figured out all this stuff and it will be great to have a script do the dirty work for us!
> 
> One of the awesome things about code review is seeing other's alternatives, so I've put some ideas here that you can take or leave. If the script is working then I know we don't want to spend too much time but I am not sure that I would be able to fix this script in the future should something break. Sometimes these things are making a gamble of how much time should we spend now to maybe avoid pain later? Sometimes its not worth it and sometimes it is. 🤷‍♀️
> 
> I think these types of conversion scripts could be very useful for us in the long run so even if we don't get this 100% polished today - towards the goal of just getting this in and material out - I could always come back to it later and maybe this could be a whole spinoff package!

Thanks for taking a look and thanks for your suggestions! I was trying to avoid dependencies like stringr based on how we were changing the previous code, but I understand your interest in readability. I guess I will try to balance both. The code is definitely working for me now ( I used it to convert all of my quizzes and tested them on coursera and they all worked), but I think it's ok to polish it a bit. Indeed, it would be exciting to make this a spinoff package!